### PR TITLE
getting around one convert to int problem

### PR DESCRIPTION
**Pull request by vivekbhr**

### CODE REVIEW

File: hicexplorer/hicFindTADs.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def parse_arguments(args=None):
     parser = argparse.ArgumentParser(
         formatter_class=argparse.RawDescriptionHelpFormatter,
         description="""
+branch : new
```
Comment (bgruening) [inline hicexplorer/hicFindTADs.py]: is this change needed?

File: hicexplorer/hicFindTADs.py [hunk 2]
```diff
@@ -919,8 +920,8 @@ def load_spectrum_matrix(file):
             fields = line.strip().split('\t')
             chrom, start, end = fields[0:3]
             chrom_list.append(chrom)
-            start_list.append(int(start))
-            end_list.append(int(end))
+            start_list.append(int(float(start)))
```
Comment (bgruening) [inline hicexplorer/hicFindTADs.py]: @vivekbhr just curious how does a string look like ...

File: hicexplorer/hicFindTADs.py (hunk 1 above)
Comment (vivekbhr) [inline hicexplorer/hicFindTADs.py]: @bgruening sorry that was my quick test after installing that I am running the right script. (due to so many python envs on my system). Removed that now

File: hicexplorer/hicFindTADs.py (hunk 2 above)
Comment (vivekbhr) [inline hicexplorer/hicFindTADs.py]: the values were like this : 0.0 , 1.0 ... but it was still reading it as string rather than float.. Then threw this error :
```
Error reading line: 1. The start field is not an integer.
Error message: invalid literal for int() with base 10: '0.0'
```
